### DISCUSSION TITLE
Does DisplayData show the data used during training with verbose=False?

### DISCUSSION THREAD

**Discussion by lenbrocki** [facebookresearch/ParlAI#4794] [Q&A]
When using DisplayData to show examples from the EmpatheticDialogue dataset it shows only text/label examples when verbose=False and additionally emotion and situation when verbose=True. 

During fine-tuning, does EmpatheticDialogue by default only return text/label pairs or additionally also the emotion and situation? If the former is the case, how can I activate returning emotions and situations as well?

**Comment by klshuster** [comment]
CC @EricMichaelSmith, as I think there might be various flags within the teachers that put these emotions/situations within the context. In general, whatever is shown with `verbose=False` will be what is shown to the agent; the other fields are sent in the observation but will need to be specially handled if you wanted to use them. But I would guess there is a teacher (`-t ...`) that includes these directly in the context for the model

**Reply by EricMichaelSmith** [reply]
Hi @lenbrocki ! Unfortunately there's no subclass of this teacher that I know of that puts the `emotion` and `situation` within the text/label pair - echoing @klshuster, I'd probably subclass this teacher and format the text/label fields to include the emotions/situations in the format that you'd like them.

**Reply by lenbrocki** [reply]
Thanks for your answers, I will try with your suggestion!
